.github/workflows/update-snapshot.yml

```diff
@@ -8,6 +8,11 @@ on:
 
 jobs:
   update_examples_snapshot:
+    strategy:
```

Comment:
Have we already separated the committed snapshots for the two flavors?

Reply:
Yes! This is already done ;D